airflow/www/decorators.py

```diff
@@ -35,51 +35,56 @@
 logger = logging.getLogger(__name__)
 
 
-def action_logging(f: T) -> T:
+def action_logging(func=None, event=None):
```

Comment:
```suggestion
def action_logging(func: Optional[T]=None, event: Optional[str]=None) -> T:
```
Seems we need typing here